Inconsistent capitalization info and typos (#154)
mongodb/docs-meta: source/style-guide/terminology/general-term-guidelines/use-consistent-terms.txt

   same document can be confusing to users and translators, so avoid
   it when possible.

-  Avoid neologisms. Examples of neologisms (newly invented words) 

>    include *marketecture* or *edutainment*. Most such words are specific 

Reviewer: ```suggestion
   include *marketecture* or *edutainment*. Most of these words are specific 
```

   same document can be confusing to users and translators, so avoid
   it when possible.

> -  Avoid neologisms. Examples of neologisms—or
>    newly invented words—are *marketecture* or *edutainment*. Most

Reviewer 2: ```suggestion
-  Avoid neologisms. Examples of neologism (that is,
   newly invented words) are *marketecture* or *edutainment*. Most
```

> -  Avoid neologisms. Examples of neologisms—or
>    newly invented words—are *marketecture* or *edutainment*. Most

Reviewer: ```suggestion
-  Avoid neologisms. Examples of neologisms
   (newly invented words) include *marketecture* or *edutainment*. Most
```
